src/packages/sdk/src/engineCommands.ts

```diff
@@ -75,42 +75,15 @@ export async function getDMMF({
       createMany: getMessage('createMany'),
       groupBy: getMessage('groupBy'),
     }
-    if (enableExperimental) {
-      enableExperimental = enableExperimental
-        .filter((f) => {
-          const removeMessage = removedFeatureFlagMap[f]
-          if (removeMessage) {
-            if (!process.env.PRISMA_HIDE_PREVIEW_FLAG_WARNINGS) {
-              console.log(removeMessage)
-            }
-            return false
-          }
-
-          return true
-        })
-        .filter(
-          (e) =>
-            ![
-              'middlewares',
-              'aggregateApi',
-              'distinct',
-              'aggregations',
-              'nativeTypes',
-              'atomicNumberOperations',
-              'createMany',
-              'groupBy',
-            ].includes(e),
-        )
-    }
 
-    const experimentalFlags =
-      enableExperimental &&
-      Array.isArray(enableExperimental) &&
-      enableExperimental.length > 0
-        ? [`--enable-experimental=${enableExperimental.join(',')}`]
-        : []
+    enableExperimental?.forEach((f) => {
```

Comment:
Would be great to rename `enableExperimental` everywhere to `previewFeatures` maybe? this old naming is confusing I think?